### ISSUE TITLE
Phantom test fail on build with Gulp

### ISSUE THREAD

**Issue by irthos** [zurb/foundation-apps#617]
Running Foundation for Apps just for the directives using Moda generator.

``` ShellSession 
PhantomJS 1.9.8 (Mac OS X) Factory: Api should be defined FAILED
	TypeError: 'null' is not an object (evaluating 'mediaQueries[key].replace')
	    at init (/Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/foundation-apps/dist/js/foundation-apps.js:630)
	    at mqInitRun (/Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/foundation-apps/dist/js/foundation-apps.js:596)
	    at invoke (/Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/angular/angular.js:4203)
	    at /Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/angular/angular.js:4025
	    at forEach (/Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/angular/angular.js:323)
	    at createInjector (/Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/angular/angular.js:4025)
	    at workFn (/Users/i/Apps/Work/JaaPharma/medviz/app/bower_components/angular-mocks/angular-mocks.js:2425)
```

Repo is https://github.com/irthos/medviz
fwiw, i wrapped the line in an 'if' to force the build.
``` javascript
    console.log(mediaQueries, "mediaqueries");
    if(mediaQueries.default === true){
        for(var key in mediaQueries) {
            mediaQueries[key] = 'only screen and (min-width: ' + mediaQueries[key].replace('rem', 'em') + ')';
        }
    }
```

**Comment by gakimball** [comment]
Fixed via a685cccf2a13dff77cdd728f41f9dbbbefbdf643. It'll be in the next patch release.

**Comment by irthos** [comment]
Cheers
